eventmesh-common/src/main/java/org/apache/eventmesh/common/config/convert/ConverterMap.java

```diff
@@ -0,0 +1,146 @@
+import org.apache.eventmesh.common.config.convert.converter.PropertiesConverter;
+import org.apache.eventmesh.common.config.convert.converter.StringConverter;
+import org.apache.eventmesh.common.config.convert.converter.URIConverter;
+
+import java.lang.reflect.Field;
+import java.net.URI;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Properties;
+import java.util.TreeMap;
+import java.util.Vector;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import inet.ipaddr.IPAddress;
+
+/**
+ * Use to map the field clazz and the converter for the field clazz
+ */
+public class ConverterMap {
+
+    public static final Logger LOGGER = LoggerFactory.getLogger(ConverterMap.class);
+
+    private static final ObjectConverter objectConverter = new ObjectConverter();
+
+    private static final Map<Class<?>, ConvertValue<?>> classToConverter = new HashMap<>();
+
+    static {
+        register(new URIConverter(), URI.class);
+        register(new EnumConverter(), Enum.class);
+        register(new DateConverter(), Date.class);
+        register(new StringConverter(), String.class);
+        register(new LocalDateConverter(), LocalDate.class);
+        register(new IPAddressConverter(), IPAddress.class);
+        register(new PropertiesConverter(), Properties.class);
+        register(new LocalDateTimeConverter(), LocalDateTime.class);
+        register(new ListConverter(), List.class, ArrayList.class, LinkedList.class, Vector.class);
+        register(new MapConverter(), Map.class, HashMap.class, TreeMap.class, LinkedHashMap.class);
+        register(new BaseDataTypeConverter.CharacterConverter(), Character.class, char.class);
+        register(new BaseDataTypeConverter.ByteConverter(), Byte.class, byte.class);
+        register(new BaseDataTypeConverter.ShortConverter(), Short.class, short.class);
+        register(new BaseDataTypeConverter.IntegerConverter(), Integer.class, int.class);
+        register(new BaseDataTypeConverter.LongConverter(), Long.class, long.class);
+        register(new BaseDataTypeConverter.FloatConverter(), Float.class, float.class);
+        register(new BaseDataTypeConverter.DoubleConverter(), Double.class, double.class);
+        register(new BaseDataTypeConverter.BooleanConverter(), Boolean.class, boolean.class);
+    }
+
+    public static void register(ConvertValue<?> convertValue, Class<?>... clazzs) {
+        for (Class<?> clazz : clazzs) {
+            classToConverter.put(clazz, convertValue);
+        }
+    }
+
+    /**
+     * Get the converter for the field
+     *
+     * @param field The field to be parsed
+     * @return the converter for the field
+     */
+    public static ConvertValue<?> getFieldConverter(Field field) {
```

Comment:
What's difference between ConvertValue and FieldConverter?

Reply:
ConvertValue is the top-level interface of the converter; FieldConverter is the converter of a specific field, for example, the converter of a field of type String is StringConverter

Reply:
It's should have a goold name for ConvertValue. @eight-nines

Reply:
Okay, I'll follow up!